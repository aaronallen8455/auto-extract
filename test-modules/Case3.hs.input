module Case3 where

z = 9

test x = let y = 12 in 1 + EXTRACT@simple (x + 3 * y + z)
