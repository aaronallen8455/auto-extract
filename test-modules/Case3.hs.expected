module Case3 where

z = 9

test x = let y = 12 in 1 + simple x y

simple :: Integer -> Integer -> Integer
simple x y = x + 3 * y + z
